docs/swagger.yaml

```diff
@@ -1606,6 +1638,36 @@ components:
       type: array
       items:
         $ref: "#/components/schemas/Deployment"
+    Prediction:
+      type: object
+      properties:
+        data:
+          type: object
+          properties:
+            names:
+              type: array
+              items:
+                type: string
+            ndarray:
+              type: array
+              items:
+                type: string

```

Comment:
@miguelfferraz use aqui o `oneOf` para contemplar os 3 tipos de body:
https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/

`{"data":{"ndarray": [ ], "names": []}}`
`{"data":{"ndarray": [ ], "names": []}}`
`{"strData":"plainText"}`
`{"binData":"base64"}`